scripts/image-builder/configure.sh

```diff
@@ -16,7 +16,7 @@ fi
 sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
     cockpit-composer bash-completion podman runc genisoimage \
     createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build \
-    containernetworking-plugins expect
+    containernetworking-plugins expect avahi-tools
```

Comment:
Why in the image-builder dependencies instead of the developer dependencies?

Reply:
Neither of the script felt like a 100% match since to me image-builder == test runner, but changed to another one (guess makes sense when you run tests from dev VM -> dev VM)